### PR TITLE
[翻译完毕] dev_install.

### PR DESCRIPTION
**Pull request by rexwangcc**
Some updates are still pending in the main repo. Let me finish the base translation and we could update later :octocat:

### CODE REVIEW

File: locales/zh_CN/LC_MESSAGES/dev_install.po [hunk 1]
```diff
@@ -3,75 +3,85 @@
...
+"(If on Ubuntu) Execute ``sudo apt install libtinfo-dev clang-8``. (``clang-7`` "
+"should work as well)."
 msgstr ""
+"（在Ubuntu下）执行 ``sudo apt install libtinfo-dev clang-8``。（ ``clang-7`` 应"
+"该同样可以工作）。"
 
 #: ../../dev_install.rst:27
 msgid "(If on other Linux distributions) Please build clang 8.0.1 from scratch:"
-msgstr ""
+msgstr "（在其他Linux发行版下）请从头构建clang 8.0.1："
```
Comment (archibate) [inline locales/zh_CN/LC_MESSAGES/dev_install.po]: Let's use `从头构建` -> `从源码构建` for clarify the process. I think `build from scratch` is not clean enough and should be fixed in en doc too. (`build from source` is better) wdyt? @yuanming-hu

File: locales/zh_CN/LC_MESSAGES/dev_install.po [hunk 2]
```diff
@@ -83,58 +93,78 @@ msgstr ""
...
+"On **Arch Linux**, you can easily install CUDA via ``pacman -S cuda`` without "
+"downloading the installer manually."
 msgstr ""
+"在 **Arch Linux** 下，你可以轻松地通过 ``pacman -S cuda`` 来安装CUDA并跳过手动下"
+"载安装器的环节。"
 
 #: ../../dev_install.rst:66
 msgid "Setting up Taichi for development"
-msgstr ""
+msgstr "为开发环境配置Taichi"
```
Comment (archibate) [inline locales/zh_CN/LC_MESSAGES/dev_install.po]:
```suggestion
msgstr "开发者的 Taichi 配置"
```

File: locales/zh_CN/LC_MESSAGES/dev_install.po [hunk 3]
```diff
@@ -83,58 +93,78 @@ msgstr ""
...
 
 #: ../../dev_install.rst:68
 msgid "Clone the taichi repo **recursively**, and build:"
-msgstr ""
+msgstr "请 **递归地** 克隆Taichi代码库，然后构建："
```
Comment (archibate) [inline locales/zh_CN/LC_MESSAGES/dev_install.po]:
```suggestion
msgstr "请 **递归地** 克隆 Taichi 代码库，然后构建："
```
Please see wiki, we left a space between cn and en, the same for all :)

**Review by archibate — COMMENTED** [review]
LGTM + nits.

File: locales/zh_CN/LC_MESSAGES/dev_install.po (hunk 1 above)
Comment (rexwangcc) [reply locales/zh_CN/LC_MESSAGES/dev_install.po]: Highly agree with using `build from source` instead!

Comment (archibate) [reply locales/zh_CN/LC_MESSAGES/dev_install.po]: Nice PR in taichi-dev/taichi#1040! Let's run `make merge` to get that update now :)

Comment (rexwangcc) [reply locales/zh_CN/LC_MESSAGES/dev_install.po]: Done! (It seems I accidently checked in some rst files and the updated taichi submodule while resolving the merge conflicts 🤦 should I revert them?)

**Review by archibate — APPROVED** [review]
thx!

**Pull request merged**